docker-compose.yml

```diff
@@ -12,17 +12,19 @@ services:
       - ./etc/lighttpd/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
       - ./:/var/www:cached
     links:
-      - enwiki-db-replica
+      - enwiki.db-replica
   watch:
     image: docker-registry.tools.wmflabs.org/toollabs-nodejs-web
     working_dir: /code/client/
     command:
       - ./bin/watch
     volumes:
       - ./:/code:cached
-  enwiki-db-replica:
+  enwiki.db-replica:
     build:
       context: .
       dockerfile: Dockerfile-replica-db
     env_file:
       - ./server/.env
+    ports:
+      - 4711:4711
```

Comment:
well, since we are using the replicas for development I thought it would be nice to be able to query the replicas through the same container. I use it to validate things. No reason to keep it other than it is handy to have

Reply:
no problem. I forget that this is only used in development.